### PR TITLE
i18n(zh-cn): Update `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update `configuration-reference.mdx`

#### Related issues & labels (optional)

- #8439

### CODE REVIEW

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+- **秘密服务端变量**：这些变量不会包含在你的最终包中，可以通过 `astro:env/server` 模块提供的 `getSecret()` 助手函数在服务端访问：
+
+    ```js
+    import { getSecret } from "astro:env/server"
+
+    const API_SECRET = getSecret("API_SECRET") // typed
+    const SECRET_NOT_IN_SCHEMA = getSecret("SECRET_NOT_IN_SCHEMA") // string | undefined
+    ```
+
+**注意：**因为没有安全的方式将数据发送到客户端，所以不支持秘密客户端变量。因此，在你的 schema 中不能同时配置 `context: "client"` 和 `access: "secret"`。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1603]:
```suggestion
**注意：**因为没有方式可以将数据安全地发送到客户端，所以不支持秘密客户端变量。因此，在你的 schema 中不能同时配置 `context: "client"` 和 `access: "secret"`。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+                API_SECRET: envField.string({ context: "server", access: "secret" }),
+            }
+        }
+    }
+})
+```
+
+目前支持三种数据类型：字符串、数字和布尔值。
+
+环境变量有三种类型，由 `context`（客户端或服务端）和 `access`（私有或公开）的组合设置在你的 [`env.schema`](#experimentalenvschema) 中确定：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1580]:
```suggestion
环境变量有三种类型，由 `context`（client 或 server）和 `access`（ private 或 public）的组合设置在你的 [`env.schema`](#experimentalenvschema) 中确定：
```

这里应该不用翻译，另外 private 似乎应该为 secret，好像英文的写错了

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 3]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+#### experimental.env.schema
+
+<p>
+
+**类型：** `EnvSchema`<br />
+**默认值：** `undefined`<br />
+<Since v="4.10.0" />
+</p>
+
+一个使用 `envField` 来定义环境变量的数据类型（`string`、`number` 或 `boolean`）及其属性的对象：`context`（客户端或服务端）、`access`（公开或秘密）、使用的默认值以及该环境变量是否为 `optional`（默认为 `false`）。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1616]:
```suggestion
一个使用 `envField` 来定义环境变量的数据类型（`string`、`number` 或 `boolean`）及其属性的对象：`context`（client 或 server）、`access`（public 或 secret）、使用的默认值 `default` 以及该环境变量是否为可选的 `optional`（默认为 `false`）。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 4]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+	headers: {
+		"Content-Type": "application/json",
+		"Authorization": `Bearer ${API_TOKEN}`
+	},
+	body: JSON.stringify({ appId: PUBLIC_APP_ID })
+})
+---
+```
+
+要定义环境变量的数据类型和属性，请在你的 Astro 配置中的 `experimental.env.schema` 声明一个 schema。`envField` 助手允许你将变量定义为字符串、数字或布尔值，并在一个对象中传递属性：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1559]:
```suggestion
要定义环境变量的数据类型和属性，请在你的 Astro 配置中用 `experimental.env.schema` 声明一个 schema。`envField` 助手函数允许你将变量定义为字符串、数字或布尔值，并在一个对象中传递属性：
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 5]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+<p>
+
+**类型：** `object`<br />
+**默认值：** `undefined`<br />
+<Since v="4.10.0" />
+</p>
+
+启用实验性的 `astro:env` 功能。
+
+`astro:env` API 允许你为环境变量配置类型安全的 schema，并指示它们是否应该在服务端或客户端可用。从适当的 `/client` 或 `/server` 模块中导入并使用你定义的变量：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1541]:
```suggestion
`astro:env` API 允许你为环境变量配置类型安全的 schema，并指示它们是否应该在服务端或客户端可用。从相应的 `/client` 或 `/server` 模块中导入并使用你定义的变量：
```

**Comment by liruifengv** [comment]
Vercel build failed. Seems to be related to https://github.com/withastro/docs/pull/8429

image

**Comment by huyikai** [comment]
> Vercel build failed. Seems to be related to #8429
> 
> image

没有调查出问题原因。能帮忙看一下吗。

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 6]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+                API_SECRET: envField.string({ context: "server", access: "secret" }),
+            }
+        }
+    }
+})
+```
+
+目前支持三种数据类型：字符串、数字和布尔值。
+
+环境变量有三种类型，由 `context`（client 或 server）和 `access`（ private 或 public）的组合设置在你的 [`env.schema`](#experimentalenvschema) 中确定：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1580]:
```suggestion
环境变量有三种类型，由 `context`（client 或 server）和 `access`（secret 或 public）的组合设置在你的 [`env.schema`](#experimentalenvschema) 中确定：
```

Comment (liruifengv) [reply src/content/docs/zh-cn/reference/configuration-reference.mdx:1580]: Update from https://github.com/withastro/docs/pull/8448/files

**Review by liruifengv — APPROVED** [review]
LGTM. Thank you!

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 7]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+- **秘密服务端变量**：这些变量不会包含在你的最终包中，可以通过 `astro:env/server` 模块提供的 `getSecret()` 助手函数在服务端访问：
+
+    ```js
+    import { getSecret } from "astro:env/server"
+
+    const API_SECRET = getSecret("API_SECRET") // typed
+    const SECRET_NOT_IN_SCHEMA = getSecret("SECRET_NOT_IN_SCHEMA") // string | undefined
+    ```
+
+**注意：**因为没有方式可以将数据安全地发送到客户端，所以不支持秘密客户端变量。因此，在你的 schema 中不能同时配置 `context: "client"` 和 `access: "secret"`。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1603]: 发现一点样式渲染问题

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 8]
```diff
@@ -1526,3 +1526,107 @@ export function onRequest(ctx, next) {
...
+    ```js
+    import { PUBLIC_PORT } from "astro:env/server"
+    ```
+
+- **秘密服务端变量**：这些变量不会包含在你的最终包中，可以通过 `astro:env/server` 模块提供的 `getSecret()` 助手函数在服务端访问：
+
+    ```js
+    import { getSecret } from "astro:env/server"
+
+    const API_SECRET = getSecret("API_SECRET") // typed
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1599]:
```suggestion
    const API_SECRET = getSecret("API_SECRET") // 定义的类型
```
这里翻译一下吧

**Review by liruifengv — APPROVED** [review]
LGTM!

**Pull request merged**